apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -19,42 +19,43 @@ export default class ProjectCard extends React.Component {
     currentGallery: PropTypes.oneOf(['personal', 'public']).isRequired,
     showFullThumbnail: PropTypes.bool,
     isDetailView: PropTypes.bool,
+    showReportAbuseHeader: PropTypes.bool,
   };
 
   constructor(props) {
     super(props);
 
     this.state = {
-      showReportAbuse: false,
-      showReportHeader: false, // may need to change this state in the future to utilize report cookies - if gallery ever keeps an immediate report
+      isShowingReportAbusePopUp: false,
+      hasBeenReported: false, // may need to change this state in the future to utilize report cookies - if gallery ever keeps an immediate report
     };
     this.showReportAbusePopUp = this.showReportAbusePopUp.bind(this);
     this.closeReportAbusePopUp = this.closeReportAbusePopUp.bind(this);
-    this.showReportedHeader = this.showReportedHeader.bind(this);
+    this.onReportAbuse = this.onReportAbuse.bind(this);
   }
 
   showReportAbusePopUp() {
     this.setState({
-      showReportAbuse: true,
```

Comment:
Most of the changes here are just renames to better distinguish from the new prop coming in (`showReportAbuseHeader`)